### ISSUE TITLE
[Enhancement] Provide flag to package without devtools

### ISSUE THREAD

**Issue by soapdog** [mozilla/qbrt#20]
## Objective
Create a flag for the ```qbrt package``` command change the packaging process to exclude the devtool from the final bundle. If we can't exclude the devtools then make so they don't launch automatically.

## Why?
This _might_ be easy to do and will be required for when the project is more mature anyway.

**Comment by mykmelez** [comment]
Indeed, it's a good idea, and we should be able to exclude them. In fact, I'd make that the default behavior (and add a flag like --with-devtools to include them). We'll just have to ensure that something sensible happens if the app calls *Runtime.openDevTools* when the devtools aren't available.

Also, note that we currently package all of browser/, not only the devtools. We shouldn't do that, per [/bin/postinstall.js#L221](https://github.com/mozilla/qbrt/blob/da0b6a2/bin/postinstall.js#L221). But that can be a separate issue.
